pom.xml

```diff
@@ -85,7 +85,7 @@ LICENSE file.
     <mongodb.version>3.0.3</mongodb.version>
     <mongodb.async.version>2.0.1</mongodb.async.version>
     <orientdb.version>2.1.8</orientdb.version>
-    <redis.version>2.0.0</redis.version>
+    <jedis.version>2.8.1</jedis.version>
```

Comment:
I changed this because it is only controlling the library version of Jedis, not Redis.  Went ahead and updated it to the latest version.